docs/source/api.rst

```diff
@@ -1,5 +1,84 @@
 AA Toolbox API
 ==============
 
-.. automodule:: aatoolbox
+Data sources
+------------
+
+CHIRPS
+^^^^^^
+
+.. automodule:: aatoolbox.datasources.chirps.chirps
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+Common Operational Datasets
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. automodule:: aatoolbox.datasources.codab.codab
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+FEWS NET
+^^^^^^^^
+
+.. automodule:: aatoolbox.datasources.fewsnet.fewsnet
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+IRI
+^^^
+
+Seasonal forecasts
```

Comment:
Maybe remove this subheading since there is only one IRI forecast for now anyway